test/scsi_defs_test.cc

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+#include "lib/scsi_defs.h"
+
+namespace {
+
+/*
+    Tests the ControlByte class
+  */
+
+  TEST(ControlByteClass, ShouldReturnCorrectControlByteCtor) {
+    uint8_t obsolete = 0b00;
+    uint8_t naca = 0b1;
+    uint8_t reserved = 0b000;
+    uint8_t vendorSpecific = 0b11;
+    uint8_t expected = 0b11000100;
+
+    scsi_defs::ControlByte *cb = new scsi_defs::ControlByte(obsolete, naca, reserved, vendorSpecific);
+    uint8_t actual = cb->getValue();
+
+    EXPECT_EQ(expected, actual);
```

Comment:
nit: I'd inline the values of expected/actual directly into this statement (and similarly below) for better clarity/concision.